utils/txHandler.ts

```diff
@@ -155,6 +155,26 @@ export const getEventResultFromMangataTx = function (
   return createEventResultfromExtrinsic(extrinsicResult as MangataGenericEvent);
 };
 
+export async function getEventErrorfromSudo(sudoEvent: MangataGenericEvent[]) {
+  const api = getApi();
+
+  const userBootstrapErr = hexToU8a(
```

Comment:
this name must be generic. not necesary be related to bootstrap errors, right?

Reply:
change to eventErrorValue